pegasus/sites.v3/code.org/public/educate/resources/videos.md

```diff
@@ -99,6 +99,14 @@ We are building a growing library of educational videos available for re-use by
 
 </div>
 
+<div class="col-50" style="float: left; padding: 10px;">
+
+<% facebook = {:u=>'https://youtu.be/LVV_93mBfSU'} %>
+<% twitter = {:url=>'https://youtu.be/LVV_93mBfSU', :related=>'codeorg', :text=>'THE INTERNET: How Search Works @codeorg'} %>
```

Comment:
I guess we use all caps for a number of video captions on this page, so it's kind of the style we've chosen.  Seems like a chance to use bold if we really want to emphasize certain words, but not a big deal to leave as-is.